test/compiler/_literal.ods

```diff
@@ -1,6 +1,7 @@
 state 42
 state 42.1
 state "forty-two"
+
```

Comment:
There is a [special circle in hell](http://www.buzzfeed.com/samir/things-that-will-drive-your-ocd-self-insane) for people like you....
![#perfectionist](http://images-cdn.moviepilot.com/image/upload/v1430129970/reynolds_rjl1hk.gif)